x-pack/filebeat/Jenkinsfile.yml

```diff
@@ -35,9 +35,17 @@ stages:
             branches: true     ## for all the branches
             tags: true         ## for all the tags
         stage: extended
-    build:
-        mage: "mage build test"
-        withModule: true       ## run the ITs only if the changeset affects a specific module.
+    unitTest:
+        mage: "mage build unitTest"
+        withModule: true
+        stage: mandatory
+    goIntegTest:
+        mage: "mage goIntegTest"
+        withModule: true
```

Comment:
I'm not sure if the withModule is required for the Go ITs

Reply:
It's also used in the goIntegTest -> https://github.com/elastic/beats/pull/19908#issuecomment-658927205 and https://github.com/elastic/beats/pull/18592#issuecomment-633618743

So I said nothing regarding the `goIntegTest` stage